lnwallet/size.go

```diff
@@ -0,0 +1,107 @@
+package lnwallet
+
+import (
+	"github.com/roasbeef/btcd/blockchain"
+)
+
+const (
+	WitnessFactor      = blockchain.WitnessScaleFactor
+	MaxTransactionCost = blockchain.MaxBlockCost / 10
```

Comment:
This variable should be renamed to `MaxTransactionCost` in order to remain consistent with the currently accepted name for this unit. Also, this isn't the _max_ weight, but it's the value used widely as max within the default policies of many full-nodes. So maybe `MaxTransactionWeightPolicy` would be an even better name.